apps/web/components/booking/pages/AvailabilityPage.tsx

```diff
@@ -133,6 +139,15 @@ const AvailabilityPage = ({ profile, plan, eventType, workingHours, previousPage
     setTimeFormat(is24hClock ? "HH:mm" : "h:mma");
   };
 
+  // Recurring event sidebar requires more space
+  const maxWidth = selectedDate
+    ? recurringEventCount
+      ? "max-w-6xl"
+      : "max-w-5xl"
+    : recurringEventCount
+    ? "max-w-4xl"
+    : "max-w-3xl";
+
```

Comment:
Had to widen up the viewport to avoid text wrapping, as the new text in the left panel takes a bit more space